centaur/src/it/scala/centaur/reporting/CromwellDatabase.scala

```diff
@@ -0,0 +1,38 @@
+package centaur.reporting
+
+import cats.effect.IO
+import com.typesafe.config.Config
+import cromwell.database.slick.{EngineSlickDatabase, MetadataSlickDatabase}
+import cromwell.database.sql.tables.{JobKeyValueEntry, MetadataEntry}
+import cromwell.database.sql.{EngineSqlDatabase, MetadataSqlDatabase}
+
+import scala.concurrent.ExecutionContext
+
+class CromwellDatabase(rootConfig: Config) {
+
+  private val cromwellConfig = rootConfig.getConfig("cromwell")
+
+  private lazy val engineDatabase: EngineSqlDatabase = EngineSlickDatabase.fromParentConfig(cromwellConfig)
+  private lazy val metadataDatabase: MetadataSqlDatabase = MetadataSlickDatabase.fromParentConfig(cromwellConfig)
```

Comment:
ToL Could take these types in the constructor and leave it up to the caller to know how to provide it.